Tests/LibWeb/Layout/expected/table/rowspan.txt

```diff
@@ -106,27 +106,27 @@ ViewportPaintable (Viewport<#document>) [0,0 800x600]
       PaintableWithLines (BlockContainer(anonymous)) [8,8 784x0]
       PaintableWithLines (TableWrapper(anonymous)) [8,8 229.359375x86]
         PaintableBox (Box<TABLE>) [8,8 229.359375x86]
-          PaintableBox (Box<TBODY>) [8,8 221.359375x76] overflow: [8,8 227.359375x84]
-            PaintableBox (Box<TR>) [10,10 221.359375x19] overflow: [10,10 225.359375x19]
+          PaintableBox (Box<TBODY>) [10,10 225.359375x82]
+            PaintableBox (Box<TR>) [10,10 225.359375x19]
               PaintableWithLines (BlockContainer<TH>) [10,10 72.046875x19]
                 TextPaintable (TextNode<#text>)
               PaintableWithLines (BlockContainer<TH>) [84.046875,10 74.515625x19]
                 TextPaintable (TextNode<#text>)
               PaintableWithLines (BlockContainer<TH>) [160.5625,10 74.796875x19]
                 TextPaintable (TextNode<#text>)
-            PaintableBox (Box<TR>) [10,31 221.359375x19] overflow: [10,31 225.359375x40]
+            PaintableBox (Box<TR>) [10,31 225.359375x19] overflow: [10,31 225.359375x40]
```

Comment:
Also here

Reply:
Also `rowspan="2"` here on the first cell
![image](https://github.com/user-attachments/assets/b11f9a7e-a0ef-4084-a898-92b09bf1443c)